dart/lib/src/attachment.dart

```diff
@@ -0,0 +1,112 @@
+import 'dart:typed_data';
+export 'attachment_extensions/attachment_extensions.dart';
+
+// https://develop.sentry.dev/sdk/envelopes/#attachment
+
+/// Arbitrary content which gets attached to an event.
+class Attachment {
```

Comment:
probably SentryAttachment to avoid import coalision

Reply:
As long as you're not dealing with emails :D

Reply:
yep, thats why `SentryAttachment`

Reply:
As already pointed out, different in the other SDKs and also [Python](https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/attachments.py). I disagree with naming it `SentryAttachment`, but it's up to you to decide. @bruno-garcia what's your opinion ⬆️?

Reply:
Dart is a bit sensitive to clashing types (though latest version had some improvements to deal with these cases).
No strong opinions here. Given the history of name clashing issues in this repo I suppose SentryAttachment is probably safer